Kajabi/pine
# [DSS-296] Tokens - Script Implementation

# Description

Adds a script that will take tokens in JSON format and convert it to CSS custom properties. This script will also create and format the `index.scss` file used for imports. 

**Note: The JSON and SCSS files included are dummy data for testing and will be removed in a future PR.**



Example of CSS custom prop being used to change the Button color:
<img width="109" alt="Screen Shot 2023-02-15 at 11 43 41 AM" src="https://user-images.githubusercontent.com/14791307/219110463-3127390e-970c-46be-b1da-ad3e75d7382f.png">
<img width="292" alt="Screen Shot 2023-02-15 at 11 43 37 AM" src="https://user-images.githubusercontent.com/14791307/219110487-4b8f6ad8-43f6-4c3f-a053-d638f6cef75e.png">


## Type of change


# How Has This Been Tested?

1. To test this locally, delete all scss files in the `libs/core/src/global/styles/tokens` folder.
2. Run `yarn install`
3. Navigate to `sage/lib/core` and run `yarn build.tokens`
4. Check that all token category files are generated from the json file in `libs/core/tokens`
5. Check that the `index.scss` file is generated with all token categories properly imported.
6. Test generated custom props on an existing live component to ensure that they work.


**Test Configuration**:

- sage versions:
- os:
- browsers:
- screen readers:
- misc:

# Checklist:
